### ISSUE TITLE
[design] Enhancement: File types supported

### ISSUE THREAD

**Issue by gauravmunjal13** [gauravmunjal13/Indexed_search#23]
I think we should open only text, doc, xls and mail files. However, for other files, searching will only search for the query in the name of those files.

**Comment by ha90** [comment]
Agreed. Whatever files that are text-only (and that can be opened by us i.e. we have the permission to open the file) should be opened.

**Comment by ha90** [comment]
I think we need to accumulate all the decisions, issues and other things that relate to design in a common place. Any ideas for this?

**Comment by gauravmunjal13** [comment]
sure! Here, raising individual issues not making things easy. As no notifiction system when you open the git. Its all via mail one knows.

Asana can be a choice. But I don't like it actually.  Following are the choices:
- Asana (nope)
- Evernote (nope)
- orange srum (seems  better)
- tupeap (nope)
- odoo (dont know)
- taiga (ok)

Looking others too for proect management.

**Comment by gauravmunjal13** [comment]
https://taiga.io/

seems better
